motionplan/solverFrame.go

```diff
@@ -9,6 +9,7 @@ import (
 	"go.viam.com/rdk/motionplan/tpspace"
 	frame "go.viam.com/rdk/referenceframe"
 	"go.viam.com/rdk/resource"
+	"go.viam.com/rdk/services/motion"
```

Comment:
`motionplan` should not depend on anything in `services`. Probably `PlanStep` should move to `motionplan` where `Plan` is also defined.

Reply:
i'll revert to how it was since we convert this type to proto and I do not think that function belongs in the motionPlan package